pest/src/error.rs

```diff
@@ -431,6 +426,18 @@ impl<R: RuleType> Error<R> {
     }
 }
 
+impl<R: RuleType> ErrorVariant<R> {
+    pub fn message(&self) -> Cow<str> {
```

Comment:
Could you please provide a doc here in a similar fashion to what other docs in the project look like?

Reply:
Could you please do it in the style of [this example](https://github.com/pest-parser/pest/blob/master/pest/src/error.rs#L194) for the sake of consistency?